vitrina/datasets/forms.py

```diff
@@ -420,6 +420,78 @@ def __init__(self, request=None, organization=None, *args, **kwargs):
         )
 
 
+class DatasetResourceForm(BaseResourceForm):
+    temporal_start = DateField(
+        widget=forms.TextInput(attrs={"type": "date"}),
+        required=False,
+        label=_("Laikotarpio pradžia"),
+        help_text=_("Duomenų rinkinio laikotarpio pradžia. Atitinka dct:temporal."),
+    )
+    temporal_end = DateField(
+        widget=forms.TextInput(attrs={"type": "date"}),
+        required=False,
+        label=_("Laikotarpio pabaiga"),
+        help_text=_("Duomenų rinkinio laikotarpio pabaiga. Atitinka dct:temporal."),
+    )
+
+    class Meta:
+        model = Dataset
+        fields = (
+            "title",
+            "description",
+            "temporal_start",
+            "temporal_end",
+            "is_public",
+            "tags",
+            "catalog",
+            "frequency",
+            "access_rights",
+            "files",
+            "name",
+            "contact",
+            "creator",
+            "publisher",
+            "managed_by_publisher",
+            "landing_page",
+        )
+
+    def __init__(self, request=None, organization=None, *args, **kwargs):
+        super().__init__(request, organization, *args, **kwargs)
+
+        self.helper.layout = Layout(
+            Field("is_public", placeholder=_("Ar duomenys vieši")),
+            Field("title", placeholder=_("Duomenų rinkinio pavadinimas")),
+            Field("name", placeholder=_("Duomenų rinkinio kodinis pavadinimas")),
+            Field("description", placeholder=_("Detalus duomenų rinkinio aprašas")),
+            inline_fields(
+                Field("temporal_start", placeholder=_("Pasirinkite pradžios datą")),
+                Field("temporal_end", placeholder=_("Pasirinkite pabaigos datą")),
+            ),
+            Field("files"),
+            Field("tags", placeholder=_("Surašykite aktualius raktinius žodžius")),
+            Field("landing_page"),
+            Field("catalog"),
+            Field("frequency"),
+            Field("access_rights"),
+            Field("contact"),
+            Field("managed_by_publisher"),
+            Field("creator"),
+            Field("publisher"),
+        )
+
+    def clean(self):
+        start = self.cleaned_data.get("temporal_start")
+        end = self.cleaned_data.get("temporal_end")
+        if start and end:
+            if start > end:
```

Comment:
These could be combined, since no more logic is in between. In case there is no `start` or no `end` the condition will not get to the `start > end` part to fail.

```suggestion
        if start and end and start > end:
```